doc/mgr/dashboard.rst

```diff
@@ -690,7 +690,11 @@ We provide a set of CLI commands to manage user accounts:
 
 - *Create User*::
 
-  $ ceph dashboard ac-user-create [--force-password] <username> [<password>] [<rolename>] [<name>] [<email>] [--enabled] [<pwd_expiration_date>]
+  $ ceph dashboard ac-user-create [--force-password] <username> [<password>] [<rolename>] [<name>] [<email>] [--enabled] [<pwd_expiration_date>] [--pwd_update_required]
```

Comment:
Does the `--pwd_update_required` flag have to be passed as the last argument? I find this ordering quite strange-looking and would suggest to reorder this example so that the options are listed first:
```suggestion
  $ ceph dashboard ac-user-create [--enabled] [--force-password]  [--pwd_update_required] <username> [<password>] [<rolename>] [<name>] [<email>] [<pwd_expiration_date>]
```

Reply:
> Does the --pwd_update_required flag have to be passed as the last argument?

No, the ordering is independent.

>  I find this ordering quite strange-looking

I agree, i did not want to change too much, but will re-arrange it.